streamwebs_frontend/streamwebs/tests/models/test_water_quality.py

```diff
@@ -119,32 +83,8 @@ def test_datasheet_ManyToOneSite(self):
     def test_datasheet_CreateWaterQuality(self):
```

Comment:
Since you have tests for both "with required fields" and "with required and optional fields", this test may be redundant.